index.html

```diff
@@ -43,7 +36,7 @@ <h1 id="title">
             <!-- Type Number section -->
             <div class="form-group">
```

Comment:
♻️ Required Changes ♻️
 Kindly make sure to give add the email section. Currently, it is not possible to input email in the survey form.
Check the comments under the review. 😀